src/Paket.Core/embedded/Paket.Restore.targets

```diff
@@ -21,16 +21,16 @@
     <PaketExePath Condition=" '$(PaketExePath)' == '' AND Exists('$(PaketToolsPath)paket')">$(PaketToolsPath)paket</PaketExePath>
     <PaketExePath Condition=" '$(PaketExePath)' == '' AND Exists('$(PaketRootPath)paket.exe')">$(PaketRootPath)paket.exe</PaketExePath>
 
-    <PaketExePath Condition=" '$(PaketExePath)' == '' AND '$(OS)' == 'Windows_NT' ">$(PaketToolsPath)paket.exe</PaketExePath>
-    <PaketExePath Condition=" '$(PaketExePath)' == '' AND '$(OS)' != 'Windows_NT' AND '$(PaketExeImage)' == 'assembly' ">$(PaketToolsPath)paket.exe</PaketExePath>
-    <PaketExePath Condition=" '$(PaketExePath)' == '' AND '$(OS)' != 'Windows_NT' AND '$(PaketExeImage)' == 'native' ">$(PaketToolsPath)paket</PaketExePath>
+    <PaketExePath Condition=" '$(PaketExePath)' == '' AND '$(OS)' == 'Windows_NT' AND Exists('$(PaketToolsPath)paket.exe')">$(PaketToolsPath)paket.exe</PaketExePath>
+    <PaketExePath Condition=" '$(PaketExePath)' == '' AND '$(OS)' != 'Windows_NT' AND '$(PaketExeImage)' == 'assembly' AND Exists('$(PaketToolsPath)paket.exe') ">$(PaketToolsPath)paket.exe</PaketExePath>
+    <PaketExePath Condition=" '$(PaketExePath)' == '' AND '$(OS)' != 'Windows_NT' AND '$(PaketExeImage)' == 'native' AND Exists('$(PaketToolsPath)paket')  ">$(PaketToolsPath)paket</PaketExePath>
```

Comment:
seems changes here will break the `PaketBootstrapperStyle == proj` style, since if we will use `.paket/paket.bootstrapper.props` [mentioned here ](https://github.com/enricosada/paket-netcore-testing-as-tool#how-to-migrate-from-paket-net ) , then this `Condition` will never matched (if `Exists('$(PaketToolsPath)paket')` then `PaketExePath` will be assigned before). Since `paket` will be installed from target `PaketBootstrapping` after this check.

The old version will cover this situation. If we want to support global paket installed like this `dotnet tool install -g Paket`, seems we should check `Exists('$(PaketToolsPath)paket.bootstrapper.proj')` not check `Exists('$(PaketToolsPath)paket')`